src/2048-expert/index.html

```diff
@@ -13,6 +13,27 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 EXPERT</div>
+        <p class="modal-text">
+            The aim of the game is to get <strong>2048</strong>
+            <br>
+            <br>
+            Use the arrow keys to move the tiles
+            <br>
+            After each move, tiles with the same value will get merged
+            <br>
+            <br>
+            Tiles will randomlly be generated in number 2 and 4
+            <br>
+            After each move two new tiles will appear on the board
```

Comment:
nit pick: put this line before "Tiles will ..." so the flow of instructions make more sense